### PR TITLE
Update coding style suggested by refurb

### PR DESCRIPTION
**Pull request by justinchuby**
### Description

Update coding style suggested by refurb.

Refurb can be added to lintrunner too but I am hoping ruff can catch up at some point because refurb is slower.

### Motivation and Context

Better code

### CODE REVIEW

**Comment by justinchuby** [comment]
@jcwchen

File: onnx/backend/test/case/node/blackmanwindow.py [hunk 1]
```diff
@@ -37,9 +39,9 @@ def export() -> None:
         a2 = 0.08
         y = a0
         y += a1 * np.cos(
-            2 * 3.1415 * np.arange(0, size, 1, dtype=np.float32) / (size - 1)
+            2 * math.pi * np.arange(0, size, 1, dtype=np.float32) / (size - 1)
         )
         y += a2 * np.cos(
-            4 * 3.1415 * np.arange(0, size, 1, dtype=np.float32) / (size - 1)
+            4 * math.pi * np.arange(0, size, 1, dtype=np.float32) / (size - 1)
```
Comment (jcwchen) [inline onnx/backend/test/case/node/blackmanwindow.py:45]: I think the output.pb will need to be updated accordingly. (`3.141592653589793` vs original `3.1415`) Please rerun test generation.

File: onnx/reference/ops/aionnxml/_common_classifier.py [hunk 2]
```diff
@@ -63,7 +65,7 @@ def erf_inv(x: float) -> float:
     if x == 0:
         return 0
     log = np.log(x)
-    v = 2.0 / (3.14159 * 0.147) + 0.5 * log
+    v = 2.0 / (math.pi * 0.147) + 0.5 * log
```
Comment (jcwchen) [inline onnx/reference/ops/aionnxml/_common_classifier.py:68]: DITTO

File: onnx/backend/test/case/node/hammingwindow.py [hunk 3]
```diff
@@ -34,6 +36,6 @@ def export() -> None:
         a0 = 25 / 46
         a1 = 1 - a0
         y = a0 - a1 * np.cos(
-            2 * 3.1415 * np.arange(0, size, 1, dtype=np.float32) / (size - 1)
+            2 * math.pi * np.arange(0, size, 1, dtype=np.float32) / (size - 1)
```
Comment (jcwchen) [inline onnx/backend/test/case/node/hammingwindow.py:39]: DITTO

File: onnx/backend/test/case/node/stft.py [hunk 4]
```diff
@@ -45,7 +47,7 @@ def export() -> None:
         a0 = 0.5
         a1 = 0.5
         window = a0 + a1 * np.cos(
-            2 * 3.1415 * np.arange(0, length, 1, dtype=np.float32) / length
+            2 * math.pi * np.arange(0, length, 1, dtype=np.float32) / length
```
Comment (jcwchen) [inline onnx/backend/test/case/node/stft.py:50]: DITTO

**Review by jcwchen — COMMENTED** [review]
Thanks for improving. Please regenerate TestCoverage.md as well.

File: onnx/backend/test/case/node/blackmanwindow.py (hunk 1 above)
Comment (justinchuby) [reply onnx/backend/test/case/node/blackmanwindow.py:45]: Done

Comment (justinchuby) [inline onnx/backend/test/case/node/blackmanwindow.py:45]: Why don't we just use `np.blackman`? Although I think this can be another pr.

**Comment by justinchuby** [comment]
Now using np.pi for $\pi$

File: onnx/backend/test/case/node/blackmanwindow.py (hunk 1 above)
Comment (justinchuby) [reply onnx/backend/test/case/node/blackmanwindow.py]: #5342

**Comment by justinchuby** [comment]
> LGTM. Thanks for updating! refurb seems good to have. I wonder how slow it is?

Similar to mypy. I can test that once this PR is merged

**Comment by justinchuby** [comment]
@xadupre could you help approve this change? Thanks!

**Review by xadupre — APPROVED** [review]

**Review by gramalingam — APPROVED** [review]
